### PR TITLE
fix: typo in `ABSTRACTION`

### PR DESCRIPTION
**Pull request by Aakodal**
Humain -> Human
energie -> energy

### CODE REVIEW

**Comment by AntoineJT** [comment]
I'm not sure about that, maybe you can do this :
```diff
- Pour comprendre l'utilité de ce principe et comment l'appliquer, prenons comme exemple une classe Humain, puisque c'est un truc qu'on connaît plutôt pas trop mal.
+ Pour comprendre l'utilité de ce principe et comment l'appliquer, prenons comme exemple une classe Human (Humain), puisque c'est un truc qu'on connaît plutôt pas trop mal.
```

**Comment by LukaMrt** [comment]
I agree with @AntoineJT . The entire code is in english so the class name must be in english too but you can mention the french translation (even it is obvious 🤔) if you prefer

File: poo/abstraction/fr/ABSTRACTION.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Abstraction
 
-L'idée de l'**abstraction**, c'est de découper une classe en morceaux, pour pouvoir partager (via un passage de paramètre dans une méthode par exemple) **un seul** de ces morceaux plutôt que l'objet "en entier". Pour comprendre l'utilité de ce principe et comment l'appliquer, prenons comme exemple une classe Humain, puisque c'est un truc qu'on connaît plutôt pas trop mal.
+L'idée de l'**abstraction**, c'est de découper une classe en morceaux, pour pouvoir partager (via un passage de paramètre dans une méthode par exemple) **un seul** de ces morceaux plutôt que l'objet "en entier". Pour comprendre l'utilité de ce principe et comment l'appliquer, prenons comme exemple une classe Human, puisque c'est un truc qu'on connaît plutôt pas trop mal.
```
Comment (AntoineJT) [inline poo/abstraction/fr/ABSTRACTION.md:3]:
```suggestion
L'idée de l'**abstraction**, c'est de découper une classe en morceaux, pour pouvoir partager (via un passage de paramètre dans une méthode par exemple) **un seul** de ces morceaux plutôt que l'objet "en entier". Pour comprendre l'utilité de ce principe et comment l'appliquer, prenons comme exemple une classe Human (Humain), puisque c'est un truc qu'on connaît plutôt pas trop mal.

```

**Comment by Tran-Antoine** [comment]
Works for me then.

**Pull request merged**